tsconfig.json

```diff
@@ -9,6 +9,7 @@
         "noUnusedLocals": true,
         "jsx": "react",
         "jsxFactory": "jsx",
+        "esModuleInterop": true,
```

Comment:
One of the imports required this. I don't even remember. Whatever. Should be on anyway.